packages/builder/src/components/design/settings/componentSettings.js

```diff
@@ -42,6 +43,7 @@ const componentMap = {
   filter: FilterEditor,
   url: URLSelect,
   columns: ColumnEditor,
+  allowedFields: BasicColumnEditor,
```

Comment:
I would prefer to use a more explicit name for this new settings type that's more in line with our naming convention - maybe something like `columns/basic`.